cmake/testing.cmake

```diff
@@ -17,8 +17,19 @@ target_link_libraries( GridToolsTest INTERFACE Threads::Threads)
 include(workaround_threads)
 _fix_threads_flags()
 
+include(FetchContent)
 option(INSTALL_GTEST OFF)
-add_subdirectory(./tools/googletest)
+mark_as_advanced(INSTALL_GTEST)
```

Comment:
Maybe you can add a TODO or make it even an explicit workaround before 3.13: INSTALL_GTEST doesn't need to be a cached variable when CMake >= 3.13:

https://cmake.org/cmake/help/v3.13/command/option.html